cmd/system-controller/system-controller.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/sh
+
+set -Ceu
```

Comment:
Can we use one or all of these set options - https://github.com/skupperproject/skupper-router/blob/main/.github/scripts/compile.sh#L23 ?

Reply:
I think we can set it to: `set -Ceufxo`, in order to keep it Posix shell compliant. WDYT?